runit-elixir/Dockerfile

```diff
@@ -3,6 +3,19 @@ FROM socrata/runit-bionic
 # Set up environment
 ENV LANG C.UTF-8
 
+# Add erlang solutions repo
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y install wget && \
+    wget https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb && \
+    DEBIAN_FRONTEND=noninteractive dpkg -i erlang-solutions_1.0_all.deb
+
+# Install erlang & elixir
+# NOTE: esl-erlang & elixir are pinned to specific versions
+# Any changes to these versions should be synchronized in the
+# jenkins-workers cookbook and a new AMI should be built.
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y install esl-erlang=1:24.1.* elixir=1.12.*
```

Comment:
Are we ok that this is gonna float to the latest available version until we're more comfort we've stabilized the build?

Reply:
Nm. Since this is a revert, it's what we've been doing all along.

Reply:
yea, i think floating is fine here. we used to let it float :shrug: